src/locales/en/translation.json

```diff
@@ -813,6 +819,34 @@
         "line_2": "Currently pool has only {{amount}} {{currency}} available.",
         "line_3": "Pool will increase once users will repay their loans."
       }
+    },
+    "poolChart": {
+      "totalLiquidity": "Total Liquidity",
+      "apy": "{{asset}} APY%"
+    },
+    "rowTable": {
+      "noLiquidityProvided": "Lend your asset and earn income",
+      "tableHeaders": {
+        "apy": "APY%",
```

Comment:
```suggestion
        "apy": "APY",
```

Reply:
No need % in this case, as the result already have %

![Captura de tela de 2021-06-01 14-05-50](https://user-images.githubusercontent.com/1178856/120364169-48391f80-c2e3-11eb-8d56-72700a4b311a.png)